README.md

```diff
@@ -52,30 +40,89 @@ Ask someone to give you the token when in doubt.
 
 This needs to be done only once.
 
-## Installing / upgrading the package itsef
+## [Requirements](#requirements)
+
+#TODO: write requirements that the web delivery needs to satisfy in order for the module to work
+
+## [How to install](#how-to-install)
+> The module is designed to run in the browser. It can be used in node as well, but the primary target is  for  the browser.
+
+### [Installation via script tag](#install-via-script-tag)
+```html
+<script src="https://unpkg.com/@forward-distribution/bridget"></script>
+<script type="text/javascript">
+  // Initialize the module. Must be called before any other methods
+  bridget.init();
+</script>
+```
+Put the above script tags in between the <head> tags of your HTML page. It will attach a global `bridget` object. You can access it either via `bridget` or `window.bridget`.
+
+## [Alternative installation via NPM](#alternative-installation-via-npm)
+The module is available on NPM & as a github package which can be used if you’re using a front-end packager like Browserify or Webpack:
 
 Just run
 
 ```
 npm install -g @forward-distribution/bridget
 ```
 
-# Development
+You can then require the lib like a standard NPM module
+
+## [How to use](#how-to-use)
+### Reacting to user actions.
+Actions are something that a user performed on your website such as “Clicked on a teaser”, "Navigated to a page/department", or “Attempted to share”.
+
+In the context of your webpage, the browser itself or your custom JavaScript code handles these user events, and reacts to the accordingly. 
 
-Requirements:
+In the mobile context, however, these actions need to be propagated and offloaded from the browser (webview) and onto the app, to be handled natively. This is where you, as an integrator, will need to call bridget with the appropriate action (see [api reference](#api)).
+
+As an example, handling a click event on the share action of an article.
+
+```js
+	onClickEvent = e => {
```

Comment:
Is this not something we want to be handled entirely through data properties as well ? For example, `bridget` deciding on its own if `bridge.shareArticle({url: slug, description: description, title: title})` is going to be invoked (or just simply injecting it into the element's `onClick` handling) or something of the sort. Just a thought, haven't really thought about a valid solution for this yet.

Reply:
I would not start with the example for sharing, but with the example for linking.

We should document what is happening when the user clicks a link. We should probably tell him that a new web-view will be opened if the link is internal, and external browser will be opened when the link is external. Then, we start explaining about the possibilities for over-rides. I.E. if you want this thing to happen when you click on a link, these are the data properties that you can append to your anchor tag.

Reply:
I should've mentioned that this is just a WIP skeleton of how we expect the integration guide to be structured. No concrete examples of how the bridge works or would work yet. We're in the process of still evaluating and testing ideas.

So, agreed on both your points. Ideally, we want to "hijack" these user interactions and the bridge decides for itself what needs to be invoked and what data it is expecting. And yes, linking should really be our first priority here, since it is the most ubiquitous.

Reply:
I would say that special cases like this, let's say click event on a div element, is something that is an integrator responsibility. And yes, we want to give them an option how they can do this, and the example is probably something like you have described here. But, from the point of the integration guide, I would still like that to be "lower" in the explanation about what is happening.

Example explanation:
1. All the navigation events happen on the anchor tags. If the anchor tag contains an internal URL, then native navigation happens. If the anchor tag contains external URL, then the external browser is opened.
2. If you want to over-ride this behaviour, then please add the following data attr to the anchor tag.
3. If you want to enhance the behaviour of navigation (native title for example), then please add the following data attr to the anchor tag.
4. If you want to manually trigger the navigation events (native nav, external browser), then you can use the following code (and here you write such an example)